packages/gatsby/cache-dir/blank.css

```diff
@@ -0,0 +1,3 @@
+.thisIsDummyRuleSoThatRspackHaveSomeInitialCSSGuaranted {
+  z-index: inherit;
+}
```

Comment:
without actual `css` content we were hitting HMR problem (404 on `css` hot update refetch)